client/X11/xf_client.c

```diff
@@ -1076,8 +1076,9 @@ static const button_map xf_button_flags[NUM_BUTTONS_MAPPED] = {
 	{ Button2, PTR_FLAGS_BUTTON3 },
 	{ Button3, PTR_FLAGS_BUTTON2 },
 	{ Button4, PTR_FLAGS_WHEEL | 0x78 },
-	{ Button5, PTR_FLAGS_WHEEL | PTR_FLAGS_WHEEL_NEGATIVE | 0x78 },
-	{ 6, PTR_FLAGS_HWHEEL | PTR_FLAGS_WHEEL_NEGATIVE | 0x78 },
+	{ Button5, PTR_FLAGS_WHEEL | PTR_FLAGS_WHEEL | 0x78 },
```

Comment:
This is now wrong. It should either be `{ Button5, PTR_FLAGS_WHEEL |  0x188 }` or `{ Button5, PTR_FLAGS_WHEEL | 0x100 - 0x78 }`.
What you did here is duplicating the `PTR_FLAGS_WHEEL` flag and ensuring that Button5 now behaves the same as Button 4, which should not be the case (it should be the opposite direction).

Reply:
@pnowack thx, sorry, pushed the wrong commit.